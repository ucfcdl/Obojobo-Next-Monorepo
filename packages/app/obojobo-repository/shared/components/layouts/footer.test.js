import React from 'react'
import renderer from 'react-test-renderer'

mockStaticDate()

// require used to make sure it's loaded after mock date
const Footer = require('./footer')

describe('Footer', () => {
	test('renders', () => {
		const component = renderer.create(<Footer />)
		const tree = component.toJSON()

		expect(tree).toMatchSnapshot()
	})
})
