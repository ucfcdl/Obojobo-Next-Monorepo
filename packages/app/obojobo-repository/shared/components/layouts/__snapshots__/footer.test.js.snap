// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer renders 1`] = `
<footer>
  <div
    className="ucf-open-footer"
  >
    <div
      className="ucf-open-image"
    >
      <a
        href="https://ucfopen.github.io"
      >
        <div
          className="ucf-open-logo"
        >
          UCF Open
        </div>
      </a>
    </div>
    <div
      className="ucf-open-desc"
    >
      Obojobo is a UCF open source project.
    </div>
    <div
      className="ucf-open-links"
    >
      <a
        href="https://ucfopen.github.io/Obojobo-Docs/"
      >
        Documentation
      </a>
      <a
        href="https://github.com/ucfopen/Obojobo"
      >
        Obojobo on Github
      </a>
      <a
        href="https://ucfopen.github.io"
      >
        About UCF Open
      </a>
    </div>
  </div>
  <div
    className="copyright"
  >
    <span
      className="copy"
    >
      Â© 
      <span
        id="copyright-date"
      >
        2016
      </span>
       
      <a
        href="https://ucf.edu"
      >
        University of Central Florida
      </a>
    </span>
  </div>
</footer>
`;
